docs/context/development/schema.json

```diff
@@ -332,6 +339,25 @@
             "items": {
               "$ref": "#/definitions/taxonomic_unit"
             }
+          },
+          "apomorphy": {
+            "type": "object",
+            "description": "For a phyloreference using an apomorphy, this field describes the character being used as the apomorphy",
+            "additionalProperties": true,
+            "properties": {
+              "definition": {
+                "description": "A description of this apomorphy."
```

Comment:
```suggestion
                "description": "A natural language description of this apomorphy."
```

Reply:
We should be specific here whether that description needs to follow some standard or not.